bruce-gordon/superpedia
# Feature saved supers

#### What does  this PR do?
- Creates components for Saved and SavedView
- Adds conditional rendering to all instances of "+" button if Character is saved
- Adds NavLink to Saved Supers button in Navbar
- Applies local storage so that saved characters persist on refresh

#### Where should the reviewer start?
- Saved directory
- SavedView directory
- Navbar.js
- Character.js
- App.js

#### How should this be manually tested?
- Pull down branch and navigate app.  Search for characters, add them to Saved, and refresh.  
- Look for button styling and check state in dev tools.

#### What are the relevant tickets?
- Closes #22 
- Closes #23 
- Closes #24 
- Closes #25 
- Closes #26 
- Closes #27
